collectors.rb

```diff
@@ -90,23 +88,26 @@ def collect_postgres(host, user, password)
   end
 end
 
-# These are hard-coded for now, but will be moved to a config file soon.
-clusters = ['ganeti']
-db_creds = [{:type => :mysql,
-             :host => 'testing-mysql',
-             :user => 'root',
-             :password => 'toor' }]
 
 c = Collectors.new
 
+# To test the collect_ganeti method, uncomment the following lines.
```

Comment:
It's unclear which of these lines should be uncommented to test the method.

Might be better to wrap that code with `if testing_collect_ganeti:` or the Ruby equivalent, and then document that flag with a comment?